web/crux/prisma/schema.prisma

```diff
@@ -70,18 +70,19 @@ model Node {
 }
 
 model Registry {
-  id              String           @id @default(uuid()) @db.Uuid
-  createdAt       DateTime         @default(now()) @db.Timestamptz(6)
-  createdBy       String           @db.Uuid
-  updatedAt       DateTime?        @default(now()) @db.Timestamptz(6)
-  updatedBy       String?          @db.Uuid
-  name            String           @db.VarChar(70)
+  id              String                 @id @default(uuid()) @db.Uuid
+  createdAt       DateTime               @default(now()) @db.Timestamptz(6)
+  createdBy       String                 @db.Uuid
+  updatedAt       DateTime?              @default(now()) @db.Timestamptz(6)
+  updatedBy       String?                @db.Uuid
+  name            String                 @db.VarChar(70)
   description     String?
   icon            String?
   url             String
   user            String?
   token           String?
   type            RegistryTypeEnum
+  namespace       RegistryNamesapaceEnum @default(unknown)
```

Comment:
Typoooo `RegistryNamesapaceEnum`

Reply:
too late, its known :|

Reply:
What do you mean?

Reply:
nvm :D we talked this earlier with @robot9706 in person, I'm working on it